Add Project Loved: Week of May 5th news post (#2230)
ppy/osu-wiki: news/2019-05-06-project-loved-week-of-may-5th.md

---
layout: post
title: "Project Loved: Week of May 5th"
date: 2019-05-06 16:00:00 +0000
---

We're a little late, but fear not, [Project Loved](https://osu.ppy.sh/community/forums/120) is still going strong! Check out the new picks that our captains have chosen, and vote for whether or not you'd like to see these maps enter the Loved category!

![](/wiki/shared/news/banners/project-loved.jpg)

Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the [Loved category](https://osu.ppy.sh/beatmapsets?s=8). From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each map's header image in this post to be taken to its voting thread.

> Starting this week, we'll list on this news post whether a beatmapset will have any of its difficulties excluded from Loved, since it's caused some confusion in the past. The captains usually only exclude difficulties if they have major problems that prevent it from being played properly, but it's also common that multi-mode mapsets will have some game modes removed since the set isn't being nominated for those modes explicitly. If any difficulties of a map aren't being included in the Loved nomination, a note will be written below the map's image here (see Ryuu to Shoujo to Decoherence for an example).

Reviewer: ```suggestion
Starting this week, we'll list on this news post whether a beatmapset will have any of its difficulties excluded from Loved, since it's caused some confusion in the past. The captains usually exclude difficulties only if they have major problems that prevent them from being played properly, but it's also common that multi-mode mapsets will have some game modes removed, since the set isn't nominated for those modes explicitly. If any difficulties of a map aren't being included in the Loved nomination, a note will be written below the map's image here (see Ryuu to Shoujo to Decoherence for an example).
```

168 BPM, 4:35 | 5.75★, 5.77★  
*description written by [fartownik](https://osu.ppy.sh/users/56917)*

ONLYWONDER is the last remaining piece without the leaderboard of n0ah's frederic map collection. This collection features four different frederic songs mapped in a very old school, gimmicky fashion. The set, similarly to all the other maps from the series, comes with two variants: an AR5, for the hardcore readers, and an AR8 for scrubs like myself. The map itself focuses on hard-to-read patterns, using weird stream angles and anti-jumps to achieve the goal, all decorated with a semi-strict, grid-style note placement. n0ah himself says this is probably the worst out of his frederic maps, but I think otherwise—it's arguably the most fun one of them all.

[![](/wiki/shared/news/2019-05-06-project-loved-week-of-may-5th/osu/3-ryuu-to-shoujo-to-decoherence.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[Kuroneko Dungeon - Ryuu to Shoujo to Decoherence](https://osu.ppy.sh/beatmapsets/94671#osu) by [ignorethis](https://osu.ppy.sh/users/27343)  
165 BPM, 1:57 | 7.08★  

> The [Hyper] and [Taiko Oni] difficulties are *not* being nominated for Loved.  

Reviewer: ```suggestion
The [Hyper] and [Taiko Oni] difficulties will *not* be nominated for Loved.  
```

Author: this news post is nominating them tho so i think present tense is right?

Reviewer: is using `be` in gerund form correct though?

Author: yeah? maybe? i don't even know the rule but this sounds normal to me

[![](/wiki/shared/news/2019-05-06-project-loved-week-of-may-5th/taiko/3-tower-of-heaven-you-are-slaves.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[Feint - Tower Of Heaven (You Are Slaves)](https://osu.ppy.sh/beatmapsets/179217#taiko) by [Kagami Yuki](https://osu.ppy.sh/users/1319686)  
175 BPM, 2:34 | 5.73★  
*description written by [iceOC](https://osu.ppy.sh/users/5482401)*

Tower of Heaven is a classic stream map that I'm sure many players hold dear to their hearts. The song's emotional melodies paired with the complex streams add to create this unique atmosphere that makes this map extremely enjoyable. I definitely recommend this for the fun patterns but also the whole experience.

[![](/wiki/shared/news/2019-05-06-project-loved-week-of-may-5th/taiko/4-flowering-night-fever.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)

[UNDEAD CORPORATION - Flowering Night Fever](https://osu.ppy.sh/beatmapsets/414317#taiko) by [Ekoro](https://osu.ppy.sh/users/284905)  
290 BPM, 3:52 | 8.24★  

> The [Electrical Freeze] difficulty is *not* being nominated for Loved.  

Reviewer: ```suggestion
The [Electrical Freeze] difficulty will *not* be nominated for Loved.  
```
